### PR TITLE
index: add go-source redirects for gonum/v1/xyz repos

### PR DESCRIPTION
**Pull request by sbinet**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,10 +3,19 @@
...
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <!-- go get imports -->
     <meta name="go-import" content="gonum.org/v1/gonum git https://github.com/gonum/gonum">
     <meta name="go-import" content="gonum.org/v1/plot git https://github.com/gonum/plot">
     <meta name="go-import" content="gonum.org/v1/netlib git https://github.com/gonum/netlib">
     <meta name="go-import" content="gonum.org/v1/hdf5 git https://github.com/gonum/hdf5">
+
+    <!-- redirection to godoc documentation -->
+    <meta name="go-source" content="gonum.org/v1/gonum https://github.com/gonum/gonum/ https://github.com/gonum/gonum/tree/master{/dir} https://github.com/gonum/gonum/blob/master{/dir}/{file}#{line}"/>
```
Comment (btracey) [inline index.html]: Two of these have an L in between {file} and {line}, the other two do not. Is that intentional?

Comment (sbinet) [reply index.html]: no. it's me managing to insert typos in a 5-lines diff...

**Comment by btracey** [comment]
Do you have an example of a website where this works? I can't find the equivalent on your go-hep website, and googling is failing me.

**Comment by sbinet** [comment]
here is my code:
 https://github.com/go-hep/go-hep.org/blob/master/main.go#L152

the documentation of the mechanism:
https://github.com/golang/gddo/wiki/Source-Code-Links

and, of course, golang.org/x, as an example:
```sh
$> curl https://golang.org/x/net/image
<!DOCTYPE html>
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
<meta name="go-import" content="golang.org/x/net git https://go.googlesource.com/net">
<meta name="go-source" content="golang.org/x/net https://github.com/golang/net/ https://github.com/golang/net/tree/master{/dir} https://github.com/golang/net/blob/master{/dir}/{file}#L{line}">
<meta http-equiv="refresh" content="0; url=https://godoc.org/golang.org/x/net/image">
</head>
<body>
Nothing to see here; <a href="https://godoc.org/golang.org/x/net/image">move along</a>.
</body>
</html>
```

**Comment by btracey** [comment]
Thank you. LGTM

**Review by btracey — APPROVED** [review]

**Comment by shenwei356** [comment]
Same problem. Waiting this to be merged :)

Right now I solve this manually:

```
go get -u github.com/gonum/floats
cd $GOPATH/src
mkdir -p gonum.org/v1
ln -s gonum.org/v1/gonum github.com/gonum
```

**Comment by kortschak** [comment]
@sbinet *ping*

**Comment by sbinet** [comment]
sorry. I don't think I have enough karma to push this.

@btracey what do you mean by "Same problem" ?

**Comment by kortschak** [comment]
>sorry. I don't think I have enough karma to push this.

Sorry. I've fixed that.

**Comment by sbinet** [comment]
note that the redirect-to-godoc won't work unless we either:
- generate the complete list of `index.html` files with content similar to https://github.com/gonum/gonum.github.io/pull/4#issuecomment-310730547 (so `gonum.org/v1/gonum/floats/index.html`, `gonum.org/v1/gonum/stat/index.html`, `gonum.org/v1/gonum/stat/index/distuv.html`, etc...) (I suppose this could be "easily" done with a simple `go run gen-tree.go`)
- or, have a (possibly Go) web server that generates them on the fly for each request (like I do [here](https://github.com/go-hep/go-hep.org/blob/cbe549a7fb970c0132466875117f2e1649bc920e/main.go#L101))

**Pull request merged**